Python/ast_unparse.c

```diff
@@ -67,6 +86,14 @@ append_repr(_PyUnicodeWriter *writer, PyObject *obj)
     if (!repr) {
         return -1;
     }
+    if ((PyFloat_CheckExact(obj) || PyComplex_CheckExact(obj))) {
+        PyObject *new_repr = replace_inf_value(repr);
```

Comment:
I assume is not a problem, but this will run `str.replace` on every annotated expression that uses floats or complex (which is pretty rare I assume). It would be good to have at least an estimate of the performance impact (which I expect to be negligible).

Reply:
I am not sure if performance matters in a specific case like this, when floats or complex numbers used as annotations. Anyway it is good to have tests but I couldn't figure out how to give future flags in pyperf, am I missing something?

Reply:
> I am not sure if performance matters in a specific case like this, when floats or complex numbers used as annotations

That is why I said `feel free to just ignore` and `(which is pretty rare I assume)` :)

I was suggesting this our of precaution, but if you find it difficult, just ignore the suggestion.